### PR TITLE
Add cli support

### PR DESCRIPTION
**Pull request by admd**

### CODE REVIEW

**Comment by admd** [comment]
@rjmateus I don't have permissions to push to this branch so please merge it.

**Comment by moio** [comment]
@admd I added you as a collaborator, but one of the first things we should do to graduate this from "PoC" to "real" project is to move it under the `uyuni-project` umbrella, can you take care of that?

**Comment by admd** [comment]
@moio sure. I will move it to the uyuni-project. Any name suggestion? hub-iss-exporter/hub-exporter?

**Comment by admd** [comment]
I don't seem to have permission to transfer this repository.

File: main.go [hunk 1]
```diff
@@ -28,20 +71,11 @@ func main() {
 		tables := schemareader.ReadAllTablesSchema(db)
```
Comment (rjmateus) [inline main.go:71]: Line 70 doesn't make sense anymore with the change in the argument processing.

File: main.go [hunk 2]
```diff
@@ -28,20 +71,11 @@ func main() {
...
-		//channelLabels := []int{102, 101, 119, 117, 103, 104, 118, 107, 108, 106} // c5
-		//channelLabels := []int{102, 103, 104, 105, 106, 107, 108} // c6
-		//channelLabels := []int{117, 118} // c7
-		//channelLabels := []int{108} // c8
-
-		//channelLabels := []int{102} // Moio's tests
+		channelLabels := parsedArgs.channleLabels
 		tableData := dumper.DumpeChannelData(db, channelLabels)
 
 		if len(os.Args) > 1 && strings.Compare(os.Args[1], "info") == 0 {
```
Comment (rjmateus) [inline main.go:77]: We should not load the argument in this way anymore. I think we should keep this logic at least to print in debug mode

**Review by rjmateus — APPROVED** [review]
Two changes but the rest are ready to go :)

**Pull request merged**